pkg/cli/x_ips.go

```diff
@@ -0,0 +1,152 @@
+// Copyright (C) 2015 Scaleway. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE.md file.
+
+package cli
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/scaleway/scaleway-cli/pkg/api"
+)
+
+var cmdIPS = &Command{
+	Exec:      runIPS,
+	UsageLine: "_ips [OPTIONS] [ARGS]",
+
+	Description: "",
+	Hidden:      true,
+	Help:        "Display your ips",
+	Examples: `
+    $ scw _ips
+    $ scw _ips IP_ID
+    $ scw _ips --new
+    $ scw _ips --update IP_ID:SERVER_ID
```

Comment:
I think, two parameters would be better than 1 parameter with a `colon` separator

@aimxhaisse what do you think ?